OpenRA.Game/CPos.cs

```diff
@@ -85,6 +85,11 @@ public MPos ToMPos(MapGridType gridType)
 			return new MPos(u, v);
 		}
 
+		public CPos AtLayer(byte layer)
```

Comment:
Can we please call this `ToLayer`?

Reply:
Yes. But it will look similar to `ToMPos`.

Reply:
Ah, thats a good point. How about `WithLayer` then.

`AtLayer` sounds like it should be returning true/false if the given layer matches...